rebound/librebound.py

```diff
@@ -64,6 +64,14 @@ def additional_forces(self,func):
             # C function pointer
             self.clibrebound.set_additional_forces_with_parameters(func)
 
+    @property
+    def post_timestep_modifications(self):
+        return self.fp   # might not be needed
+
+    @post_timestep_modifications.setter
+    def post_timestep_modifications(self,func):
+        self.clibrebound.set_post_timestep_modifications(func)
```

Comment:
I think you want to add the same logic as for the `additional_forces` setter, i.e. allow python function to be set too. Note that you'll need to keep a references to the function pointer (for `additional_forces` this is `self.fp`).